Containerfile.compute_worker_podman_gpu

```diff
@@ -0,0 +1,66 @@
+FROM fedora:37
+
+# Include deps
+RUN curl -s -L https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo | sudo tee /etc/yum.repos.d/cuda.repo && \
+    curl -s -L https://nvidia.github.io/nvidia-docker/rhel9.0/nvidia-docker.repo | sudo tee /etc/yum.repos.d/nvidia-docker.repo && \
```

Comment:
@dtuantran Not sure how this built for you, `sudo` is not setup in the container, anyway these steps run as root so sudo is not needed. I will try to fix it up.